splitgraph/commandline/cloud.py

```diff
@@ -676,9 +676,19 @@ def load_c(remote, readme_dir, repositories_file, limit_repositories):
                 rest_client.upsert_external(
                     repository.namespace, repository.repository, repository.external, credential_map
                 )
-            if repository.metadata:
-                metadata = _prepare_metadata(repository.metadata, readme_basedir=readme_dir)
-                gql_client.upsert_metadata(repository.namespace, repository.repository, metadata)
+
+    logging.info("Uploading metadata...")
+    namespace_list = []
+    repository_list = []
+    metadata_list = []
+    for repository in repositories:
+        if repository.metadata:
+            namespace_list.append(repository.namespace)
+            repository_list.append(repository.repository)
+
+            metadata = _prepare_metadata(repository.metadata, readme_basedir=readme_dir)
+            metadata_list.append(metadata)
+    gql_client.bulk_upsert_metadata(namespace_list, repository_list, metadata_list)
```

Comment:
Can you output some kind of a notification here like "updated metadata for %s repositories" (there's a function https://github.com/splitgraph/splitgraph/blob/master/splitgraph/core/output.py#L22 for pluralization)?